frontend-react/src/AppRouter.tsx

```diff
@@ -249,6 +263,14 @@ export const appRoutes: RouteObject[] = [
                     },
                 ],
             },
+            {
+                path: "/product/release-notes",
+                index: true,
+                lazy: lazyRouteMarkdown("content/product/release-notes"),
```

Comment:
This should be content/about/release-notes

Reply:
Nice catch, bad merge on my part.